support-frontend/assets/components/errorMessage/errorMessage.tsx

```diff
@@ -1,32 +1,31 @@
 // ----- Imports ----- //
+import { css } from '@emotion/react';
+import { error, textSans } from '@guardian/source-foundations';
 import type { ReactNode } from 'react';
-import SvgExclamation from 'components/svgs/exclamation';
 
 // ---- Types ----- //
+
 type PropTypes = {
-	showError?: boolean;
-	message: string | null | undefined;
+	message: string;
 	svg?: ReactNode;
 };
 
 // ----- Component ----- //
-export default function ErrorMessage(props: PropTypes): JSX.Element | null {
-	if (props.showError && props.message) {
-		return (
-			<div className="component-error-message">
-				{props.svg}
-				<span className="component-error-message__message">
-					{props.message}
-				</span>
-			</div>
-		);
-	}
 
-	return null;
+export default function ErrorMessage(props: PropTypes): JSX.Element {
+	return (
+		<div css={styles.container}>
+			{props.svg}
+			<span>{props.message}</span>
+		</div>
+	);
```

Comment:
This component is only being used in the contribs direct debit form. I simplified it just for that particular use case and deleted all of the sass and replaced it with some simple emotion